CONTRIBUTING.md

```diff
@@ -12,10 +12,11 @@ See [INSTALL.md](INSTALL.md) for instructions on setting up a running dev
 instance of dendrite, and [CODE_STYLE.md](CODE_STYLE.md) for the code style
 guide.
 
-We use `gb` for managing our dependencies, so `gb build` and `gb test` is how
-to build dendrite and run the unit tests respectively. Be aware that a list of
-all dendrite packages is the expected output for all tests succeeding with `gb
-test`. There are also [scripts](scripts) for [linting](scripts/find-lint.sh)
+As of May 2019, we're not using `gb` anymore, which is the tool we had been
+using for managing our dependencies, and switched to Go modules. To build
```

Comment:
First sentence is a bit of a run-on. I'd suggest putting a period after "dependencies" and then "Instead, we switched to Go modules".